src/schema/metadata/ScatterFraction.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: ScatterFraction
+description: |
+  Scatter fraction for each frame (Units: 0-100%).
+type: number
```

Comment:
In the spec, this is an array of numbers. https://bids-specification--774.org.readthedocs.build/en/774/04-modality-specific-files/09-positron-emission-tomography.html#pharmaceuticals

Reply:
Good catch. I must have mistranslated that one.

Reply:
Addressed in c13df8d.